build/DependencyVersions.props

```diff
@@ -74,6 +74,7 @@
     <BuildTasksFeedToolVersion>2.1.0-prerelease-02430-04</BuildTasksFeedToolVersion>
     <VersionToolsVersion>$(BuildTasksFeedToolVersion)</VersionToolsVersion>
     <DotnetDebToolVersion>2.0.0-preview2-25331-01</DotnetDebToolVersion>
+    <MicrosoftAspNetCoreMvcVersion>2.1.1</MicrosoftAspNetCoreMvcVersion>
```

Comment:
Rename this to `MicrosoftAspNetCoreMvcPackageVersion` so future patches will automatically get updated by ProdCon.

Reply:
@natemcmaster 
ProdCon does not have a "MicrosoftAspNetCoreMvcPackageVersion" per @mmitche

Reply:
It doesn't for the current patch release, but it will in future patches. ProdCon only generates this version variable if a package is produced as a part of aspnet outputs. When we produce new versions, you will want to ensure you align with the version from ProdCon.

Reply:
@peterhuene 
Can you weigh-in on this issue?

Reply:
@natemcmaster  is right for this.  You want it to be PackageVersion so that it can be overridden.